src/rgb/contract/mod.rs

```diff
@@ -63,7 +67,7 @@ pub mod test_helpers {
         );
     }
 
-    pub fn test_confidential<T>(data: &[u8], commitment: &[u8]) -> Result<T, Error>
+    pub(crate) fn test_confidential<T>(data: &[u8], commitment: &[u8]) -> Result<T, Error>
```

Comment:
So far `0x36` doesn't seem to be matching with any enum flag. Do you want this to be changed randomly? In that case, I think there's a fair amount of chance (its an u8) that it will collide with an enum flag.

Reply:
Right, you are changing the first byte, which is the enum. Ok, let's leave it as is